### ISSUE TITLE
Extended Data

### ISSUE THREAD

**Issue by MMCXXVI** [kaistomics/DeepNeo#1]
Good day!

In your paper, you compared the impact of DeepNeo-MHC, NetMHCpan, and MixMHCpred on MHC binding prediction using different **extended data** collected from multiple publications, as shown in **Figure 1 c and d**.

However, I noticed that I couldn't find any annotations or download links for these **extended data** in the Supplementary information of your paper, the webserver page, or this repository, because it is too trivial to share, or did I miss it somewhere?

Thank you a lot.

Cheers,
guangfu xue

**Comment by malachig** [comment]
In the DeepNEO paper:

```
Code availability
Codes for implementing prediction of peptide-MHC binding and prediction of immunogenicity are deposited at https://github.com/kaistomics/DeepNeo.
```

https://www.nature.com/articles/s41588-022-01273-y

The authors have publicly attested that they will share this code.  And it did indeed exist at this location previously, but has apparently now been removed.

**Comment by MMCXXVI** [comment]
> In the DeepNEO paper:
> 
> ```
> Code availability
> Codes for implementing prediction of peptide-MHC binding and prediction of immunogenicity are deposited at https://github.com/kaistomics/DeepNeo.
> ```
> 
> https://www.nature.com/articles/s41588-022-01273-y
> 
> The authors have publicly attested that they will share this code. And it did indeed exist at this location previously, but has apparently now been removed.


Indeed, two months ago, this GitHub repository contained the relevant model code but lacked the dataset files. However, at present, the repository has been completely emptied, with even the associated code being deleted. If other researchers are unable to reproduce this work, coupled with the fact that the repository was cleared by the author, there are legitimate reasons to question the validity of this research.

**Comment by ValMystletainn** [comment]
They may think github as a regular cloud storage. But it doesn't. so in April, there were the commit messsage to show how they upload the code and delete them after the paper published in commit logs.
However, the commit log can be overwrited by `git push -f `(as the do it now). Besides, issue can be complete deleted easily. So the question of the validty can be just in vain in this issue.

However, I will raise the question to the authors about the current version code released.
1. What's your code to reproduce the training process? everyone want to reproduce your work need it. we need to clarify what reasonable(for instance, some augmentation not mentioned) and unreasonable(for instance, leak testing set) tricks are used and 
2. Is your data preprocess right? data from *Properties of MHC class I presented peptides that enhance immunogenicity* in your dataset doesn't not match the data from original paper. You extend several HLA types of those immunogenic peptide in original paper where just one specific HLA type with the peptide is immunogenic.
3. Why you just released the code of v1 but not v2. and your web server only provide the v2 serve? for the trade secret? for the unknown tricks?
4. By the way. in v2 paper you said **DeepNeo-tcr has relatively small dimensions with large filter size, requiring all the nodes to have weights. Thus, we replaced ReLU function with linear function to include all feature importance of the convolution layers. ** please turn back your linear algebra course, it will tell your the compose all any finite linear/affine transformations is linear/affine transformations. And the conv layer is a affine transformation. So when you do so. you just shrink your network function sapce into a 1 conv layer net function space. A fundmental error on your work.

I just forked your code in 05/31/2023 UTC 00:14 to make a snapshot. Remember, git is a distributed version control software. When you choose to release the repo in public, everyone can get exactly same and valid copy with all history versions on their own space.
